src/client/lazy-app/Compress/Modal/index.tsx

```diff
@@ -0,0 +1,148 @@
+import { h, Component } from 'preact';
+import * as style from './style.css';
+import 'add-css:./style.css';
+import { linkRef } from 'shared/prerendered-app/util';
+import {
+  InfoIcon,
+  DiamondStarIcon,
+  ModalErrorIcon,
+} from 'client/lazy-app/icons';
+
+interface Props {}
+
+export interface ModalMessage {
+  type: 'info' | 'error' | 'update';
```

Comment:
I can't really think of reasons other than these 3 for displaying a modal - so I thought it might be better to abstract away the Icon behind the `type` parameter

Reply:
The benefit of providing the icon is unused icons won't be bundled